QModManager/API/SMLHelper/Handlers/CustomFishHandler.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace QModManager.API.SMLHelper.Handlers
+{
+    using System;
+    using System.Collections.Generic;
+    using QModManager.API.SMLHelper.Assets;
+    using System.IO;
+    using QModManager.API.SMLHelper.Interfaces;
+
+    /// <summary>
+    /// Class to manage registering of fish into the game
+    /// </summary>
+    public class CustomFishHandler : ICustomFishHandler
+    {
+        public static ICustomFishHandler Main { get; } = new CustomFishHandler();
```

Comment:
Some public members don't have XML comments.

Reply:
Is this specifically about the `CustomFishHandler` class or about all the various `Main` singleton fields?

Reply:
Neither. Generally speaking